modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,223 @@
+... In the *Expression* box, type `ovnkube_clustermanager_num_egress_ips` and click *Run queries*. If you have egress firewall resources, they are returned in the *Expression* box.
+
+. Check your cluster for multicast enabled namespaces. You can only do this by using the `oc` CLI.
+
+.. To locate namespaces with multicast enabled, enter the following command:
++
+[source,terminal]
+----
+$ oc get namespace -o json | jq -r '.items[] | select(.metadata.annotations."k8s.ovn.org/multicast-enabled" == "true") | .metadata.name'
+----
++
+.Example output
++
+[source,terminal]
+----
+namespace1
+namespace3
+----
+
+.. To describe each multicast enabled namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc describe namespace <namespace>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         my-namespace
+Labels:       kubernetes.io/metadata.name=my-namespace
+              pod-security.kubernetes.io/audit=restricted
+              pod-security.kubernetes.io/audit-version=v1.24
+              pod-security.kubernetes.io/warn=restricted
+              pod-security.kubernetes.io/warn-version=v1.24
+Annotations:  k8s.ovn.org/multicast-enabled: true
+              openshift.io/sa.scc.mcs: s0:c25,c0
+              openshift.io/sa.scc.supplemental-groups: 1000600000/10000
+              openshift.io/sa.scc.uid-range: 1000600000/10000
+Status:       Active
+----
++
+Ensure that multicast functionality is correctly configured and working as expected in each namespace. For more information, see "Enabling multicast for a project".
+
+. Check your cluster's network policies. You can only do this by using the `oc` CLI.
+
+.. To obtain information about network policies within a namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc get networkpolicy -n <namespace>
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME              POD-SELECTOR   AGE
+allow-multicast   app=my-app     11m
+----
+
+.. To provide detailed information about the network policy, enter the following command:
++
+[source,terminal]
+----
+$ oc describe networkpolicy allow-multicast -n <namespace>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         allow-multicast
+Namespace:    my-namespace
+Created on:   2024-07-24 14:55:03 -0400 EDT
+Labels:       <none>
+Annotations:  <none>
+Spec:
+  PodSelector:     app=my-app
+  Allowing ingress traffic:
+    To Port: <any> (traffic allowed to all ports)
+    From:
+      IPBlock:
+        CIDR: 224.0.0.0/4
+        Except: 
+  Allowing egress traffic:
+    To Port: <any> (traffic allowed to all ports)
+    To:
+      IPBlock:
+        CIDR: 224.0.0.0/4
+        Except: 
+  Policy Types: Ingress, Egress
+----
++
+Ensure that the behavior of the network policy is as intended. Optimization for network policies differ between SDN and OVN-K, so users might need to adjust their policies to achieve optimal performance for different CNIs. For more information, see "About network policy".
```

Comment:
I do see a handful of other instances of "OVN-K" throughout the repo, but it makes me wonder. A [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc) or SSG entry saying whether this shortname is permissible would be nice.

Also is "SDN" by itself OK here? Or should it be more specifically:
```suggestion
Ensure that the behavior of the network policy is as intended. Optimization for network policies differ between OpenShift SDN and OVN-K, so users might need to adjust their policies to achieve optimal performance for different CNIs. For more information, see "About network policy".
```